pkg/engine/imageVerify.go

```diff
@@ -66,21 +100,23 @@ func VerifyAndPatchImages(policyContext *PolicyContext) (*response.EngineRespons
 
 		logger.V(3).Info("processing image verification rule", "ruleSelector", applyRules)
 
+		var err error
+		ruleImages, err := requireValidation(policyContext, rule)
+		if err != nil {
+			appendError(resp, rule, fmt.Sprintf("failed to extract images: %s", err.Error()), response.RuleStatusError)
+			continue
+		}
+		if len(ruleImages) == 0 {
+			appendError(resp, rule, "skip run verification for image", response.RuleStatusSkip)
```

Comment:
Hmm...why do we append an error here?

Reply:
shall I rename `appendError` to `appendErrorOrSkip`?

Reply:
I see.  Perhaps we can rename to `appendResponse` or similar to be clear? And should we add the image reference in the message?